manifests/init.pp

```diff
@@ -117,9 +117,7 @@
   Optional[Hash] $cas                      = undef,
 ) {
   # Is the package cron or systemd.timer based?
-  if ($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'],'18.04') <= 0 ) or
-  ($facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'],'10') <= 0 ) or
-  ($facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'7') <= 0 ) {
+  if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'7') <= 0 {
```

Comment:
we could also drop EL7 since that's really dead soonish

Reply:
true, but that we should do in a seperate PR :)